.travis.yml

```diff
@@ -10,7 +10,7 @@ services:
 
 env:
   global:
-    - HOST_PORT=3000
+    - PORT=3000
     - DB_HOSTNAME=mongo
```

Comment:
If `DB_HOSTNAME` was removed, can we instead use `MONGO_URI` here? I think a value of `mongodb://mongo:27017` should work because the DB container is in the same docker configuration (and network).

I removed TravisCI's checks from being required, but if this makes them pass again then it would be cool as well, and a small fix to try.